responsive-domain-search/responsive-domain-search.js

```diff
@@ -22,7 +22,7 @@ document.addEventListener("DOMContentLoaded", function() {
     
     /* 
      *  Created at 10 July 2018 by PlanetCloud.
-     *  Last modified at 02 November 2018 by PlanetCloud.
+     *  Last modified at 16 March 2019 by Dog2puppy.
```

Comment:
Please keep the changelog here, you could put it like this:
```
    /* 
     *  Created at 10 July 2018 by PlanetCloud.
     *  Modified at 02 November 2018 by PlanetCloud.
     *  Last modified at 16 March 2019 by Dog2puppy.
     *  DO NOT REMOVE CREDITS!
     *  Created for: VPT (VistaPanel Team).
     */
```

Reply:
Fixed.

Reply:
Pardon? How is it fixed? Where did you change it?

Reply:
Fixed now.

Reply:
GitHub must've not saved it then